filament/src/driver/opengl/OpenGLDriver.cpp

```diff
@@ -2735,6 +2716,29 @@ void OpenGLDriver::blit(TargetBufferFlags buffers,
                 mask, GL_LINEAR);
         enable(GL_SCISSOR_TEST);
         CHECK_GL_ERROR(utils::slog.e)
+
+        // In a sense, blitting to a texture level is similar to calling setTextureData on it; in
+        // both cases, we update the base/max LOD to give shaders access to levels as they become
+        // available.
+        GLTexture* dtexture = d->gl.color.texture;
+        if ((mask & GL_COLOR_BUFFER_BIT) && dtexture) {
+            uint8_t baseLevel = dtexture->gl.baseLevel;
+            uint8_t maxLevel = dtexture->gl.maxLevel;
+            uint8_t targetLevel = d->gl.colorLevel;
+            if (targetLevel < baseLevel || targetLevel > maxLevel) {
+                GLenum target = dtexture->gl.target;
+                bindTexture(MAX_TEXTURE_UNITS - 1, target, dtexture, dtexture->gl.targetIndex);
+                activeTexture(MAX_TEXTURE_UNITS - 1);
```

Comment:
Note that I'm using the MAX-1 texture stage for mutating the texture object only because the pre-existing code did it this way, I don't totally understand why we use this particular texture slot.

Reply:
It probably helps with debugging or to try and minimize state changes. Probably doesn't matter either way.